qbft/decided.go

```diff
@@ -28,18 +28,20 @@ func (c *Controller) UponDecided(msg *SignedMessage) (*SignedMessage, error) {
 	}
 
 	// did previously decide?
-	inst := c.InstanceForHeight(msg.Message.Height)
-	prevDecided := inst != nil && inst.State.Decided
+	inst, err := c.InstanceForHeight(msg.Message.Height)
+	prevDecided := err == nil && inst != nil && inst.State.Decided
 
 	// Mark current instance decided
-	if inst := c.InstanceForHeight(c.Height); inst != nil && !inst.State.Decided {
+	if inst, err := c.InstanceForHeight(c.Height); inst != nil && !inst.State.Decided {
```

Comment:
error needs to be checked independently

Reply:
same as above, we want to failed all process cause of it?